content/en/network_performance_monitoring/installation.md

```diff
@@ -308,11 +308,14 @@ services:
 ```
 
 [1]: https://app.datadoghq.com/account/settings#api
+{{% /tab %}}
+{{% tab "ECS" %}}
+Please see specific setup instructions on our [AWS ECS][7] documentation page.
```

Comment:
```suggestion
To set up on AWS ECS, see the [AWS ECS][7] documentation page.
```

Reply:
need to add the 7 link at the bottom of the page too. should look like this:

[7]: /integrations/amazon_ecs/?tab=awscli#network-performance-monitoring-collection

Reply:
Thanks so much for going over this and the notes!